src/QuadTwistRequestLimiter.cpp

```diff
@@ -0,0 +1,91 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Twist Limiter
+//
+// Limits the rate of change of a twist and min/max values
+//
+////////////////////////////////////////////////////////////////////////////
+
+#include "iarc7_motion/QuadTwistRequestLimiter.hpp"
+#include <cmath>
+
+using namespace Iarc7Motion;
+QuadTwistRequestLimiter::QuadTwistRequestLimiter(Twist minTwist, Twist maxTwist, Twist maxChange) :
+minTwist_(minTwist) ,
+maxTwist_(maxTwist) ,
+maxTwistChange_(maxChange)
+{
+
+}
+
+// TODO sanity check these values
```

Comment:
Not doing that anymore, they aren't being fixed and it clutters the build output unnecessarily.